security_overview/data_encryption.sp

```diff
@@ -0,0 +1,43 @@
+benchmark "security_overview_data_encryption" {
+  title         = "Data Encryption"
+  description   = "All data stored in Snowflake is transparently encrypted using a key hierarchy (with cloud HSM backed root of trust), which provides enhanced security by encrypting individual pieces of data using a different key."
+  documentation = file("./security_overview/docs/data_encryption.md")
+  children = [
+    control.security_overview_data_encryption_use_tri_secret_secure,
+    control.security_overview_data_encryption_automatic_cmk_rotation,
+    control.security_overview_data_encryption_tri_secret_secure_in_database_replication,
+    control.security_overview_data_encryption_enable_periodic_rekeying,
+    control.security_overview_data_encryption_use_builtin_encryption_functions,
```

Comment:
In an IAM control, we use `built_in` instead of `builtin`, I think it should be `built_in` here since it's a hyphenated word